benchmarks/lockhammer/scripts/test_lockhammer.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+import sys
+import os
+import sh
+import platform
+import unittest
+
+ALL_TESTS = []
+
+
+def find_lh_tests():
+    for fname in os.listdir("../build"):
+        if fname.startswith("lh_"):
+            ALL_TESTS.append(fname)
+
+def test_generator_default(lhBin):
+    def test(self):
+        testBin = os.path.join("..", "build", lhBin)
+        cmd = sh.Command(testBin)
+        out = str(cmd(_err_to_out=True))
+        self.assertRegex(out, "ns elapsed", "Cannot find 'ns elapsed' keywords.")
+    return test
+
+def test_sequencer_default(className):
+    find_lh_tests()
+    for lhBin in ALL_TESTS:
+        test_lh = test_generator_default(lhBin)
+        setattr(className, "test_" + lhBin, test_lh)
```

Comment:
Should we create an explicit map of inputs to test?

Reply:
Yes, I'll add them.